.github/workflows/build-whisper-docker.yml

```diff
@@ -0,0 +1,57 @@
+#
+name: Build and publish whisper docker image
+
+# Configures this workflow to run every time a change is pushed to the branch called `release`.
+on:
+  workflow_dispatch:
+  push:
+    branches: ['main']
+    paths:
+      - "whisper_container/Dockerfile"
+      - ".github/workflows/build-whisper-docker.yml"
+
+# Defines two custom environment variables for the workflow. These are used for the Container registry domain, and a name for the Docker image that this workflow builds.
+env:
+  REGISTRY: ghcr.io
```

Comment:
Might want to think about using AWS's image repository since you're copying the entire model into the container - the bandwidth between ECR and your eventual EC2 machine should be much better than to ghcr.

Reply:
ah great shout, will do

Reply:
Though actually with https://github.com/guardian/amigo/pull/1397 the impact of this latency will be reduced as we'll be 'baking' the container into the AMI

Reply:
Doesn't this mean you require an amigo build every time you release a new version? Do we have the infra to coordinate that? Wouldn't it be easier to pull the docker image when you start the image so you don't need to extend the CI pipeline? The EC2 machine will need to pull the data either way, either as an XXL sized AMI or as a docker image.

Reply:
Sorry, I don't actually have the data - do we see higher bandwidth when pulling AMIs than in the final runtime EC2 VM? 

I guess it would be possible that they throttle down the network controller once your machine is up and running based on the instance type - and downloading the AMI is done full tilt.

Reply:
If AMIs are mounted as a network device and AWS has them available "hot" (e.g. the hypervisor can just connect to an AMI with no further loading) then this would indeed be a lot faster. 

It's surprisingly difficult to find out the internals of how AMIs are loaded onto an EC2 instance.